client/project.go

```diff
@@ -210,6 +210,7 @@ type ProjectResponse struct {
 	ResourceConfig                       *ResourceConfigResponse     `json:"resourceConfig"`
 	NodeVersion                          string                      `json:"nodeVersion"`
 	Crons                                *ProjectCronsResponse       `json:"crons"`
+	RollingRelease                       *RollingRelease             `json:"rollingRelease"`
```

Comment:
I don't think we are reading it from the project  endpoint, so this change isn't needed anymore?